themes/default/content/what-is/what-is-hashicorp-vault.md

```diff
@@ -0,0 +1,87 @@
+---
+title: What is Hashicorp Vault?
+meta_desc: |
+     Discover the essentials of HashiCorp Vault, the tool for securing, storing, and tightly controlling access to tokens, passwords, certificates, and encryption keys. Learn how Vault's secrets management and data protection safeguard sensitive data in modern cloud and DevOps environments.
+
+type: what-is
+page_title: "What is Hashicorp Vault"
+---
+
+HashiCorp Vault is a tool for secure secrets management, essential for storing and safeguarding sensitive data like API keys and passwords. It's one of many options in the ecosystem of secrets and configuration management tools supported by [Pulumi ESC](https://www.pulumi.com/product/esc/). As the complexity of modern cloud infrastructure has grown, so has the need for robust and scalable secrets management solutions. Vault's integration with Pulumi ESC addresses this need by providing a secure, centralized platform for managing sensitive information across multiple cloud environments. By leveraging Vault's capabilities within Pulumi ESC and [Pulumi Cloud](https://www.pulumi.com/product/pulumi-cloud/) infrastructure as code solutions, developers can ensure that their cloud infrastructure is secure and compliant with best practices, even as the underlying systems, complexity and requirements evolve.
+
+In this article, we'll cover the key features of [HashiCorp Vault](https://www.hashicorp.com/products/vault), why secret management is important and share the use cases of how Vault integrates into various cloud architectures, enhancing the security and management of sensitive data in complex cloud environments.
+
+### Key Features of HashiCorp Vault?
+
+Docker Configs are a resource in Docker for storing non-sensitive information such as configuration files, separate from a service's image or running containers within [Docker Swarm](https://docs.docker.com/engine/swarm/) environments. This enables keeping Docker images as generic as possible without the need for bind-mounting configuration files into containers or using environment variables. Unlike [Docker Secrets](/what-is/what-are-docker-secrets), Docker Configs are not encrypted at rest and are directly mounted into the container's filesystem.
```

Comment:
Did we take copy from the other page and not update it?

Reply:
fixed